lib/one_gadget/fetchers/objdump.rb

```diff
@@ -31,7 +31,12 @@ def extra_options=(options)
       # @param [Integer] stop The end address.
       # @return [String] The CLI command to be executed.
       def command(start: nil, stop: nil)
-        cmd = [bin, '--no-show-raw-insn', '-w', '-d', *@options, @file]
+        # --dwarf-start=0 is to make sure `suppress_bfd_header` is true to eliminate the information that includes the file path:
```

Comment:
lib/one_gadget/fetchers/objdump.rb:34:121: C: Layout/LineLength: Line is too long. [129/120] ([https://rubystyle.guide#max-line-length](https://rubystyle.guide/#max-line-length))
        # --dwarf-start=0 is to make sure `suppress_bfd_header` is true to eliminate the information that includes the file path:

Reply:
Ok, done :)
I moved some details to #204, hopefully this problem has been resolved now.